bin/ruby-build

```diff
@@ -701,7 +701,7 @@ fix_rbx_irb() {
 }
 
 require_java7() {
-  local version="$(java -version 2>&1 | grep 'java version' | head -1)"
+  local version="$(java -version 2>&1 | grep '\(java\|openjdk\) version' | head -1)"
```

Comment:
This does not work for me (alpine linux, idk 1.7.0_79)

When I do:
`java -version`

I get this result:
```
java version "1.7.0_79"
Java(TM) SE Runtime Environment (build 1.7.0_79-b15)
Java HotSpot(TM) 64-Bit Server VM (build 24.79-b02, mixed mode)
```

But when I do:
`java -version 2>&1 | grep '\(java\|openjdk\) version' | head -1`

I get no result.